scripts/verify-gofmt.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Why we are wrapping gofmt?
+# - ignore files in vendor direcotry
+# - gofmt doesn't exit with error code when there are errors
+
+GO_FILES=$(find . -path ./vendor -prune -o -name '*.go' -print)
+
+for file in $GO_FILES; do
+	gofmtOutput=$(gofmt -l "$file")
+	if [ "$gofmtOutput" ]; then
+		errors+=("$gofmtOutput")
+	fi
+done
+
+
+if [ ${#errors[@]} -eq 0 ]; then
+	echo "gofmt OK"
+else
+	echo "gofmt ERROR - These files are not formated by gofmt:"
+	for err in "${errors[@]}"; do
+		echo "$err"
+	done
+	exit 1
```

Comment:
I'd propose a simpler solution, fewer lines of code and get rid of the bashisms:
```
err_files=$(find . -path ./vendor -prune -o -name '*.go' -print | xargs gofmt -l)

if [ -z "$err_files" ]; then
    echo "gofmt OK"
else
    echo "gofmt ERROR - These files are not formated by gofmt:"
    echo "$err_files"
    exit 1
fi
```

Reply:
I don't know if it is my env or what, but that snippet fails for me.